### ISSUE TITLE
How to export symbol PaAsio_ShowControlPanel

### ISSUE THREAD

**Issue by stefanyovev** [PortAudio/portaudio#975]
Is there any configuration or modification to export this function into the built static lib ?
How are the exported symbols controlled?

(I tried modifying portaudio.def.in and then use cmake with no success.)

**Comment by RossBencina** [comment]
> Is there any configuration or modification to export this function into the built static lib ?

No. We don't support or promote that. Is there a reason why you can't use `Pa_GetStreamTime()` ?
